src/net.h

```diff
@@ -51,6 +51,8 @@ static const bool DEFAULT_WHITELISTFORCERELAY = false;
 static const int TIMEOUT_INTERVAL = 20 * 60;
 /** Run the feeler connection loop once every 2 minutes or 120 seconds. **/
 static const int FEELER_INTERVAL = 120;
+/** Run the chain-sync connection loop once every 5 minutes. **/
+static const int CHAIN_SYNC_INTERVAL = 300;
```

Comment:
can we make this mockable from the beginning?

Reply:
If #19869 is merged first, then I'll update the code here as well when I rebase.